app/common/lib/bookmarkFoldersUtil.js

```diff
@@ -117,11 +118,32 @@ const isMoveAllowed = (folderList, sourceDetail, destinationDetail) => {
   return true
 }
 
+const buildFolder = (folderDetails, folders) => {
+  folderDetails = makeImmutable(folderDetails)
+  let key = folderDetails.get('folderId')
+
+  if (!folderDetails.has('folderId')) {
+    key = getNextFolderId(folders)
+  }
+
+  return makeImmutable({
+    title: folderDetails.get('title'),
+    folderId: Number(key),
+    key: key.toString(),
+    parentFolderId: ~~folderDetails.get('parentFolderId', 0),
```

Comment:
ditto not sure which you prefer but either `~~` or `Number`

Reply:
I prefer `~~`, but please check this comment https://github.com/brave/browser-laptop/pull/10325#discussion_r137261789